exocom-server/features/support/steps-then.ls

```diff
@@ -14,6 +14,15 @@ module.exports = ->
     @verify-sent-calls {client-name, message-name, response-to: '111'}, done
 
 
+  @Then /^ExoCom broadcasts replies to the "([^"]+)" service with the same session ID$/, (client-name, done) ->
+    @verify-equal-session-id {client-name, "users.created", response-to: '111'}, done
+
+
+  @Then /^ExoCom broadcasts replies to the "([^"]+)" service with a new session ID$/, (client-name, done) ->
+    @verify-equal-session-id {client-name, "users.created", response-to: '111'}, ~>
+      @verify-new-session-id {client-name, "users.created", response-to: '111'}, done
```

Comment:
Does it make a little more sense if if we `@verify-new-session-id` first and then `@verify-equal-session-id`?